frontend/public/components/monitoring/alertmanager/alertmanager-config.tsx

```diff
@@ -550,26 +549,18 @@ export const AlertmanagerConfig: React.FC = () => {
           </div>
         </PrimaryHeading>
       </div>
-      <ul className="co-m-horizontal-nav__menu">
-        <li
-          className={classNames('co-m-horizontal-nav__menu-item', {
-            'co-m-horizontal-nav-item--active': url === configPath,
-          })}
-        >
-          <Link to={configPath} data-test-id="horizontal-link-details">
-            {t('public~Details')}
-          </Link>
-        </li>
-        <li
-          className={classNames('co-m-horizontal-nav__menu-item', {
-            'co-m-horizontal-nav-item--active': url === YAMLPath,
-          })}
-        >
-          <Link to={YAMLPath} data-test-id="horizontal-link-yaml">
-            {t('public~YAML')}
-          </Link>
-        </li>
-      </ul>
+      <NavBar
+        pages={[
+          {
+            name: t('public~Details'),
+            href: configPath,
+          },
+          {
+            name: t('public~YAML'),
+            href: YAMLPath,
+          },
+        ]}
+      />
```

Comment:
there is some issue rendering `Details` and `YAML` tab, tabs will disappear when resizing

https://github.com/user-attachments/assets/e85d3354-2a6b-4551-beff-1739562e1464

Reply:
Looks like PF6 issue to me.

Reply:
@logonoff I think the `xl` breakpoint, in the `inset` prop, might be adding too much padding on larger screens, causing the tabs to overflow sooner. Could you try using smaller breakpoints like `lg` or `md` to se it the issue persists ?

Reply:
@nicolethoen could you please assist here ?

Reply:
I fixed this issue by wrapping all Tabs in another div